spec/services/github_handler_spec.rb

```diff
@@ -81,5 +81,22 @@
         }.to change(PullRequest, :count).by(0)
       end
     end
+
+    describe 'review removal request' do
+      let!(:pull_request) { create :pull_request, github_id: 1000 }
+      let!(:user) {
+        create :user,
+          github_id: 1001,
+          login: 'pentacat',
+          node_id: 'MDExOlB1bGxc5MTQ3NDM3'
+      }
+      let!(:review_requests) { ReviewRequest.create(
```

Comment:
`do ... end` blocks are preferred for multi-line `let` variables. Our linters will report this once we add them to the project.